apis/networking/ports.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package networking
+
+// The ports we setup on our services.
+const (
```

Comment:
I created and equivalent of this [file](https://github.com/knative/serving/blob/master/pkg/apis/networking/ports.go), just in `github.com/knative/pkg` because there was a circular dependency. Perhaps some ports might be added here later? 
Sure I can add it to profiling/server but don't we have the apis/networking package as the common place for defining ports?

Reply:
Regarding the input parameter, this would complicate the calls in all binaries right now. I suppose the only place where we might want to change the port is the queue-proxy (due to port conflicts with the user application). And I'd do this (simple) change when it's actually required.

Reply:
OK. Thanks for the review. Moved the port to profiling/server.go then.